frontend/packages/console-app/src/components/guided-tours/GuidedTours.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { PageHeading } from '@console/internal/components/utils';
+import { getGuidedToursWithStatus } from './utils/guided-tour-utils';
+import GuidedTourCatalog from './GuidedTourCatalog';
+
+const GuidedTours: React.FC = () => {
+  return (
+    <>
+      <Helmet>
+        <title>Guided Tours</title>
+      </Helmet>
+      <div className="odc-empty-state__title">
```

Comment:
Remove unrelated `className`.
You can probably remove this `div` altogether?

Reply:
removed